frontend/javascripts/oxalis/model/sagas/dataset_saga.ts

```diff
@@ -31,3 +32,29 @@ export function* watchMaximumRenderableLayers(): Saga<void> {
 
   yield* takeEvery(["WK_READY", "UPDATE_LAYER_SETTING", "UPDATE_DATASET_SETTING"], warnMaybe);
 }
+
+let downsamplingWarningDidShow = false;
+export function* watchZ1Downsampling(): Saga<void> {
+  function* maybeShowWarning(): Saga<void> {
+    const currentRes = yield* select((state) => getCurrentResolution(state));
+    const currentZoomStep = yield* select((state) => state.flycam.zoomStep);
```

Comment:
```suggestion
    const currentZoomStep = yield* select((state) => state.flycam.zoomStep);
    if (currentZoomStep < 1) {
      // If the user has zoomed into the data, the rendering quality is expected to be relative low.
      // Don't show any warnings in that case.
      return;
    }
```


otherwise, zooming in very far (which also results in minecraft-quality) would show the warning even though there's nothing to do about it.